### PR TITLE
Fix homepage

### PR DESCRIPTION
**Pull request by akshaaatt**
- Homepage cleanup and footer separation

### CODE REVIEW

File: listenbrainz/webserver/templates/footer.html [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+<!--                    <img src="{{ url_for('static', filename='img/listenbrainz-logo.svg') }}" width="180" alt="ListenBrainz"/>-->
+<!--                </h3>-->
+<!--                <br/>-->
+<!--                <p class="color-gray">-->
+<!--                    ListenBrainz keeps tracks of what music you listen to and provides you with insights into your listening habits.-->
+<!--                    <br>-->
+<!--                    You can use ListenBrainz to track your listening habits, discover new music with personalized recommendations, and share your musical taste with others using our visualizations.-->
+<!--                </p>-->
+<!--                <ul class="list-unstyled">-->
+<!--                    <li class="color-a">-->
```
Comment (MonkeyDo) [inline listenbrainz/webserver/templates/footer.html:242]: I suppose all this commented code should be removed?

File: listenbrainz/webserver/templates/navbar.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-white" role="navigation">
+<nav class="navbar fixed-top navbar-expand-lg navbar-light bg-white border-bottom border-" role="navigation">
```
Comment (MonkeyDo) [inline listenbrainz/webserver/templates/navbar.html:1]: The last `border-` class doesn't look right

File: listenbrainz/webserver/templates/index/index.html [hunk 3]
```diff
@@ -43,61 +43,57 @@ <h2>Discover new music</h2>
...
+        </div>
+        <div class="panel-footer" style="text-align: center">
+          <small>Updated every 5 minutes</small><br />
+          <small><a href="{{ url_for('index.current_status') }}">more on the current status page</a></small>
         </div>
       </div>
-
     </div>
-
     <div class="col-md-5"
```
Comment (MonkeyDo) [inline listenbrainz/webserver/templates/index/index.html:97]: I suppose this will be coming in another PR, but I just wanted to mention since we're looking at the homepage that the stats and news panels look like they're missing css styles:
image

File: listenbrainz/webserver/templates/footer.html [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="col-sm-12 col-md-3">
+                <h3 class="ms-3">
+                    Fellow Projects
+                </h3>
+                <ul class="list-style-type">
+                    <li class="pt-2 pb-2">
```
Comment (MonkeyDo) [inline listenbrainz/webserver/templates/footer.html:45]: I think there's an error here, there's no MusicBrainz but ListenBrainz is here which we don't need.
I also think that means that maybe (?) you can remove the `meb-icons/ListenBrainz.svg` file you added in this PR (if it's only used in the footer)

File: listenbrainz/webserver/templates/footer.html [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+                            community.metabrainz.org
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="col-sm-12 col-md-3">
+                <h3 class="ms-3">
+                    Fellow Projects
+                </h3>
+                <ul class="list-style-type">
```
Comment (MonkeyDo) [inline listenbrainz/webserver/templates/footer.html:44]: Not sure what styles would need to be transferred from before, but I quite liked the chevrons you used in the footer:
image

File: listenbrainz/webserver/templates/footer.html (hunk 1 above)
Comment (akshaaatt) [reply listenbrainz/webserver/templates/footer.html:242]: Right!

File: listenbrainz/webserver/templates/footer.html (hunk 5 above)
Comment (akshaaatt) [reply listenbrainz/webserver/templates/footer.html:44]: Bitmap left a reply suggesting to me that we shouldn't add extra styles and hence I should remove the chevrons and keep things standard. I do think that makes sense, hence made the move. Let me know if that if fine.

File: listenbrainz/webserver/templates/footer.html (hunk 4 above)
Comment (akshaaatt) [reply listenbrainz/webserver/templates/footer.html:45]: Thanks! Fixed.

File: listenbrainz/webserver/templates/index/index.html (hunk 3 above)
Comment (akshaaatt) [reply listenbrainz/webserver/templates/index/index.html]: Fixed :)

File: listenbrainz/webserver/templates/footer.html (hunk 5 above)
Comment (MonkeyDo) [reply listenbrainz/webserver/templates/footer.html:44]: Or perhaps removing the list style entirely?

image

**Comment by MonkeyDo** [comment]
For this PR as well, I'd be happy to see some footer-specific PR.

Since we have redesigned LB front-end completely a lot of things in this PR aren't relevant anymore.
